dspace/config/spring/api/virtual-metadata.xml.openaire4

```diff
@@ -0,0 +1,448 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<beans xmlns="http://www.springframework.org/schema/beans" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns:util="http://www.springframework.org/schema/util"
+    xsi:schemaLocation="http://www.springframework.org/schema/beans http://www.springframework.org/schema/beans/spring-beans.xsd http://www.springframework.org/schema/util http://www.springframework.org/schema/util/spring-util.xsd"
+    default-lazy-init="true">
+
+    <bean class="org.dspace.content.virtual.VirtualMetadataPopulator">
+        <property name="map">
+            <!-- This map contains keys and value-refs to Map objects
+                 The keys will define which relationship type will add virtual metadata
+                 to its relationship's item and the map will define which virtual metadata fields
+                 are to be added to the item that has a relationship with this relationship type-->
+            <map>
+                <entry key="isAuthorOfPublication" value-ref="isAuthorOfPublicationMap"/>
+                <entry key="isOrgUnitOfPublication" value-ref="isOrgUnitOfPublicationMap"/>
+                <entry key="isPersonOfProject" value-ref="isPersonOfProjectMap"/>
+                <entry key="isOrgUnitOfProject" value-ref="isOrgUnitOfProjectMap"/>
+                <entry key="isOrgUnitOfPerson" value-ref="isOrgUnitOfPersonMap"/>
+                <entry key="isJournalVolumeOfIssue" value-ref="isJournalVolumeOfIssueMap"/>
```

Comment:
Since this is a separate virtual metadata configuration, I would prefer to keep it limited to the openaire-only config

Reply:
I've just removed the unused parts.